readthedocs/search/utils.py

```diff
@@ -153,24 +155,27 @@ def _indexing_helper(html_objs_qs, wipe=False):
                 delete_objects_in_es.delay(**kwargs)
 
 
-def _remove_newlines_from_dict(highlight):
-    """
-    Recursively change results to turn newlines into periods.
+def _get_inner_hits_highlights(hit, logging=False):
```

Comment:
It seems like maybe we don't even need this function. It's mostly just checking if the hit has a highlight and returning it.

Reply:
Remvoed. :+1: